src/components/focusManager.js

```diff
@@ -1,5 +1,11 @@
-define(['dom', 'scrollManager'], function (dom, scrollManager) {
-    'use strict';
+/* eslint-disable indent */
+
+import dom from 'dom';
+import scrollManager from 'scrollManager';
+
+/* eslint-disable no-unused-expressions */
+'use strict';
```

Comment:
You can remove this since ES6 modules are strict by default.

And thanks for contributing to the Jellyfin Project.

Reply:
I was not sure if that was a babel specific thing or not. did not want to make it more complex than it needed to be. 
will fix this ! :-) 

No problem! Love the software and really think if we put some effort in to it we can have the best opensource media experience!

Reply:
Thanks! Every PR, gets us closer to that goal 😀.